content/10.diagnostics.md

```diff
@@ -87,7 +87,10 @@ Most importantly, they would not work on a patient who has fully recovered from
 In this context, immunity tests are significantly more informative.
 Additionally, they can help scientists to understand why the disease has a different course among patients, as well as what strategy might work to manage the spread of the infection.
 Furthermore, serological tests hold significant interest at present because they can provide information relevant to advancing economic recovery and allowing reopenings.
-For instance, people that have developed antibodies can plausibly return to work prior to the others, based on (still-unproven) protective immunity [@doi:10.1101/2020.04.14.20065771], and if extensive enough, herd immunity which will prevent further diffusion of the virus.
+For instance, people that have developed antibodies can plausibly return to work prior to the others, based on (still-unproven) protective immunity [@doi:10.1101/2020.04.14.20065771]. 
+On a related note, for some infectious agents an epidemic can be stopped from growing through herd immunity in which enough of the population is immune to infection through vaccination and/or prior experience with infection.
+A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R0)) fraction of the population must be immune or, equivalently, less than (1/R0) fraction of the population susceptible.
+However, for SARS-CoV-2 and COVID-19, because of the calculated R0 and mortality and the recorded per-region prevalence and mortality, relying on herd immunity without vaccines and/or proven treatment options and/or strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.
```

Comment:
```suggestion
However, for SARS-CoV-2 and COVID-19, because of the calculated R0 and mortality and the recorded per-region deaths and accumulated cases with the estimated factor of undetected cases, relying on herd immunity without vaccines and/or proven treatment options and/or strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.
```